### DISCUSSION TITLE
Interesting Bug for Gateway & CP

### DISCUSSION THREAD

**Discussion by ivelliosy** [jilleb/MQB-FPA#28] [Exploration and research]
Hey all. Today I discovered that if there is a unit with Component Protection on Can, Gateway is giving CP on error as usual, but my dataset which should remember the setting I left driving profile is "not" remembering and going back to normal 😄 Interesting. Module which has a CP is a hacked 5F. I think there is no way around to bypass that. Fyi.

**Comment by batal131** [comment]
What car do you have?

**Reply by ivelliosy** [reply]
It is A3 8V.
